zerver/lib/actions.py

```diff
@@ -2754,6 +2905,7 @@ def do_send_confirmation_email(invitee, referrer):
 
 @statsd_increment("push_notifications")
 def handle_push_notification(user_profile_id, missed_message):
+    # type: (int, Dict[str, str]) -> None
```

Comment:
Oops. `missed_message['message_id']` is an `int`, but I don't know anything about other keys. I'm typing it as `Dict[str, Any]` unless I get more info.